packages/runtime/test/transport/identityDeletionPeerProcessing.test.ts

```diff
@@ -64,4 +72,116 @@ describe("IdentityDeletionProcess", () => {
         const updatedRelationship = (await services2.transport.relationships.getRelationship({ id: relationshipId })).value;
         expect(updatedRelationship.peerDeletionInfo).toBeUndefined();
     });
+
+    test("messages with multiple recipients should fail if one of the recipients has an active IdentityDeletionProcess", async () => {
+        await services1.transport.identityDeletionProcesses.initiateIdentityDeletionProcess();
+
+        await syncUntilHasEvent(services2, PeerToBeDeletedEvent, (e) => e.data.id === relationshipId);
+        await services2.eventBus.waitForRunningEventHandlers();
+
+        const result = await sendMessageToMultipleRecipients(services2.transport, [services1.address, services4.address]);
```

Comment:
Use `services3.address` instead of `services4.address` here, because there is only a Relationship between `services2` and `services3` and not between `services2` and `services4`. Then, you can simplify by removing the variable `services4` from the file.

Reply:
yea it is before I combined the errorMessage (so you could see that to one you have no relationship and the other is toBeDeleted etc) but this wasn't desired, so now I can refactor this.